include/eblob/blob.h

```diff
@@ -690,6 +690,14 @@ int eblob_start_defrag(struct eblob_backend *b);
  */
 int eblob_start_defrag_level(struct eblob_backend *b, enum eblob_defrag_state level);
 
+/*!
+ * eblob_start_defrag_in_dir() - overload version of eblob_start_defrag with defrag-level and chunks_dir params.
+ * @b - eblob backend instance
+ * @level - level of defragmentation
+ * @chunks_dir - directory for temporary chunks (overrides value specified in config)
```

Comment:
Does it override once, or finally replaces that parameter in backend config?

Reply:
It overrides chunks_dir only for the triggered defrag.